bonus.md

```diff
@@ -0,0 +1,64 @@
+
+---
+
+### Bonus Round : Oceanic Navigation
+
+For SAUVC 2024, there is a bonus round with a single task called Oceanic Navigation. This was created to allow teams allow teams to experience the challenges of operating in an oceanic environment. The bonus round will be held in the [TCOMS Ocean Basin](https://www.tcoms.sg/), a cutting edge facility for testing and developing marine technology. The basin is located at the [National University of Singapore](https://www.nus.edu.sg/).
+
+![arena](img/ocean-basin.jpg)
+<p class="image-caption"> TCOMS Ocean basin <br> <span style="font-size:0.8em"> Photo credit : <a href="https://www.facebook.com/191030801533400/photos/pb.100064187163543.-2207520000/1083302985639506">TCOMS</a></span>  </p>
+
+
+The aim of this task is for the AUV to swim through a gate in the middle of the pool. At the Ocean Basic, the AUVs will have to deal with waves and currents. Only the top teams from the final round will be invited to the bonus round.
+
+**A orange flare** may be in located anywhere in the oceanic basin. The AUV **has to avoid** touching the orange flare. Any part of the AUV touching the orange flare causes an immediate <span class="important">abort</span> of the attempt.
+
+#### Arena (Ocean Basin)
+
+The basin is equipped with a wave generator capable of generating waves up to 0.5m in height and a current generator capable of generating currents up to 1.5m/s. The wave and current generators may be activated during the bonus round.
+
+![arena](img/arena-ocean-basin.jpg)
+<p class="image-caption"> Top view of the ocean basin arena. </p>
+
+
+Since the basin is a research facility, and the area is not open to the public, only a small number of the team members will be allowed to enter the basin. The rest of the SAUVC participants will have to watch the attempt from the viewing gallery. Details on the number of team members allowed to enter the basin will be released at a later date.
+
```

Comment:
We can only enter the 'basin' itself when it is raised, and I imagine the floor would be lowered that day which means no one will be allowed to enter the basin (except divers maybe).

I think the best wording here would be:
"Since the basin is a research facility, and the area is not typically open to the public, only a
small predetermined number of the participants/viewers will be allowed into the facility to watch the attempt from the viewing gallery. Members of the team doing the attempt will have to operate at the AUV Deployment/Retrieval corner under the supervision of SAUVC committee members. SAUVC Committee members and judges and TCOMS staff will be positioned on the subcarriage in the facility which will be positioned above the gate, providing a direct view of the gate.

Details on the number of participants allowed to enter the viewing gallery will be released at a later date."